packages/unenv-preset/package.json

```diff
@@ -21,32 +21,31 @@
 	},
 	"license": "MIT OR Apache-2.0",
 	"sideEffects": false,
+	"type": "module",
 	"exports": {
 		".": {
-			"types": "./dist/index.d.ts",
-			"require": "./dist/index.cjs",
-			"import": "./dist/index.mjs"
+			"types": "./dist/src/index.d.ts",
+			"import": "./dist/src/index.js"
 		},
 		"./runtime/*": {
-			"types": "./dist/runtime/*.d.ts",
-			"require": "./dist/runtime/*.cjs",
-			"import": "./dist/runtime/*.mjs"
+			"types": "./dist/src/runtime/*.d.ts",
+			"import": "./dist/src/runtime/*.js"
```

Comment:
I highly recommend to use `.mjs` extension for tooling compatibility. `.js` at **least** has performance overhead that tools need to travese and read closest `package.json` only to find about right format.

Reply:
Is there a way to output `.mjs` with `tsc`? (I checked but didn't find a way)

Reply:
AFAIK there was a recent pr they introduced to rewrite paths. still `/index` is not auto fixed (tsc does not adheres ESM all the times -- that's the major issue and why mkdist process is safer)